qml/pages/FirstPage.qml

```diff
@@ -590,20 +629,66 @@ Page {
 
             menu: ContextMenu {
                 hasContent: lastPostNumber > 0 || !loadedMore
+                MenuLabel { text: !!watchlevel[notification_level] ? watchlevel[notification_level].name : ""; visible: (notification_level >=0) }
                 MenuItem { text: qsTr("Mark as read")
                     visible: lastPostNumber > 0 && lastPostNumber < highest_post_number
                     onDelayedClick: {
                         postCountConfig.setValue(topicid, highest_post_number);
                         lastPostNumber = highest_post_number;
                     }
                 }
+                /*
+                MenuLabel { text: ""; height: sl.height
+                    Slider{ id: sl
+                        width: parent.width
+                        stepSize: 1
+                        minimumValue: 0
+                        maximumValue: 3
+                        value: notification_level
+                        valueText: watchlevel[sliderValue].name
+                        label: qsTr("Tracking Level")
+                    }
+                }
+                */
+                MenuItem { text: watchlevel[0].action // "Mute"
+                    visible: (notification_level >= 0) && (notification_level != 0)
+                    onDelayedClick: {
+                        // muting hides the topic completely, with no way in the app to undo.

```

Comment:
There is this https://forum.sailfishos.org/latest.json?state=muted so we could always just add an option in main pulldown menu for muted topics and load it to allow unmuting, not sure how much this would be needed